### PR TITLE
INTYGFV-14430: Changed font size to be smaller and fixed mandatory ic…

### PR DESCRIPTION
**Pull request by mhornfeldt**
…on placement

### CODE REVIEW

**Comment by perfli** [comment]
Is there a reason as to why we diverge from the style guide regarding default font size for webcert? From my perspective that goes against the purpose of a style guide.

**Comment by mhornfeldt** [comment]
> Is there a reason as to why we diverge from the style guide regarding default font size for webcert? From my perspective that goes against the purpose of a style guide.

You can read more in the Jira but to summarize, test believes that the application would benefit from smaller text. The style guide is mainly developed for websites with less content and information on one single page than WC. We have suggested to the developers of the style guide a setting where components, and in this case text, could have configurable sizes. This has not been implemented yet so we have overrided some elements in the style guide. This change would override the text, previous changes have for example made input fields and buttons smaller.

**Review by perfli — CHANGES_REQUESTED** [review]

File: packages/webcert/src/components/header/WebcertHeaderUnit.tsx [hunk 1]
```diff
@@ -10,11 +10,11 @@ import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
 const Wrapper = styled.div`
   display: flex;
   align-items: center;
-  font-size: 13px;
```
[the remark this answers is not in the document]
Comment (mhornfeldt) [reply packages/webcert/src/components/header/WebcertHeaderUnit.tsx:13]: With the change of inactive unit the caregiver name had gotten a smaller font-size than the doctor name but they should be the same size. If the inactive unit text should be smaller then I can add that seperately and not on the wrapper.

[the remark this answers is not in the document]
Comment (perfli) [reply packages/webcert/src/components/header/WebcertHeaderUnit.tsx:13]: I see. As long as the inactive unit text is 13px then it's fine.

[the remark this answers is not in the document]
Comment (mhornfeldt) [reply packages/webcert/src/components/header/WebcertHeaderUnit.tsx:13]: I will change it to 12px so it matches the doctor role and unit name like before.

File: packages/common/src/components/utils/Accordion.tsx [hunk 2]
```diff
@@ -73,7 +76,7 @@ const Accordion: React.FC<Props> = ({
       return (
         <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
           <Icon iconType={icon ? icon : ''} includeTooltip={includeIconTooltip} size={iconSize} />
-          <MandatoryIcon display={displayMandatory as boolean} />{' '}
+          <MandatoryIcon additionalStyles={mandatoryIconAdditionalStyles} display={displayMandatory as boolean} />{' '}
```
[the remark this answers is not in the document]
Comment (mhornfeldt) [reply packages/common/src/components/utils/Accordion.tsx:79]: We can utilize this moving forward.

**Review by perfli — APPROVED** [review]

**Pull request merged**